jmix-email/email-flowui/src/main/resources/io/jmix/emailflowui/view/sendingmessage/sending-message-view.xml

```diff
@@ -50,6 +50,9 @@
     </data>
     <facets>
         <dataLoadCoordinator auto="true"/>
+        <settings>
```

Comment:
take a look at how we can define autoSize for the DatGrid columns + at this [issue](https://github.com/jmix-framework/jmix/issues/1925).